Makefile

```diff
@@ -2,6 +2,7 @@ default: docker-run
 
 setup:
 	(cd ui; yarn)
+	docker-compose build
 
```

Comment:
bit of an unrelated note but the ReadMe should be` make docker-database `not `make docker-postgres`